app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -83,17 +83,15 @@
 
     @components.error_summary(form.errors)
 
-    @components.section_header(messages("declaration.fiscalInformation.header"))
-
-    @components.page_title(Some(messages("declaration.additionalFiscalReferences.title")))
+    @components.heading(
+        Some(messages("declaration.fiscalInformation.header")),
```

Comment:
Would be more clear if you used `sectionHeader = Some(...)`

Reply:
Yes it would! :) Done